corehq/apps/domain/forms.py

```diff
@@ -1485,6 +1490,14 @@ def process_subscription_management(self):
         new_plan_version = DefaultProductPlan.get_default_plan_by_domain(
             self.domain, edition=self.cleaned_data['software_plan_edition'],
         )
+        kwargs = {
+            'auto_generate_credits': True,
+            'date_end': self.cleaned_data['end_date'],
+            'do_not_invoice': False,
+            'internal_change': True,
```

Comment:
This part of that ticket
> hi
> I thought we had talked as well about changing the "subgroup" email to which the subscription changes email get sent, to differentiate whether it is changed by a Dimagi person vs. an external person? 

> Maybe we haven't talked about it but would be good to implement so that some persons (like Neal) can only follow external activity?

> Right now, it seems like everything is sent to accounts+subchange and we discussed about creating another email that people can filter out with accounts+subchange+internal - when the change is done by a Dimagi person?